.circleci/config.yml

```diff
@@ -136,7 +136,7 @@ workflows:
   weekly-valgrind:
     triggers:
       - schedule:
-          cron: "0 0 * * 1"
+          cron: "30 8 * * 5"
```

Comment:
I think it would be good to write what this means as a comment:

```
# run valgrind tests at 08:30 on Friday.
# https://crontab.guru/#30_8_*_*_5
cron: "30 8 * * 5"

```